packages/cli/src/cmds/validator/keymanager/impl.ts

```diff
@@ -60,6 +60,28 @@ export class KeymanagerApi implements Api {
     );
   }
 
+  async listGraffiti(pubkeyHex: string): ReturnType<Api["listGraffiti"]> {
+    return {data: {pubkey: pubkeyHex, graffiti: this.validator.validatorStore.getGraffiti(pubkeyHex)}};
+  }
+
+  async setGraffiti(pubkeyHex: string, graffiti: string): Promise<void> {
+    this.checkIfProposerWriteEnabled();
+    this.validator.validatorStore.setGraffiti(pubkeyHex, parseFeeRecipient(graffiti));
```

Comment:
`parseFeeRecipient` on the graffiti looks wrong

Reply:
yea sending in a test file with corrections